PhoneNumberKit/Resources/PhoneNumberMetadata.json

```diff
@@ -1 +1 @@
n": "2(?:0[0-57]|7[568])\\d{4}"}},{"id": "PH","countryCode": "63","internationalPrefix": "00","nationalPrefix": "0","availableFormats": {"numberFormat": [{"pattern": "(\\d)(\\d{5})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": "2","format": "$1 $2"},{"pattern": "(\\d{4})(\\d{4,6})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": ["3(?:23|39|46)|4(?:2[3-6]|[35]9|4[26]|76)|544|88[245]|(?:52|64|86)2","3(?:230|397|461)|4(?:2(?:35|[46]4|51)|396|4(?:22|63)|59[347]|76[15])|5(?:221|446)|642[23]|8(?:622|8(?:[24]2|5[13]))"],"format": "$1 $2"},{"pattern": "(\\d{5})(\\d{4})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": ["346|4(?:27|9[35])|883","3469|4(?:279|9(?:30|56))|8834"],"format": "$1 $2"},{"pattern": "(\\d)(\\d{4})(\\d{4})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": "2","format": "$1 $2 $3"},{"pattern": "(\\d{2})(\\d{3})(\\d{4})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": "[3-7]|8[2-8]","format": "$1 $2 $3"},{"pattern": "(\\d{3})(\\d{3})(\\d{4})","nationalPrefixFormattingRule": "$NP$FG","leadingDigits": "[89]","format": "$1 $2 $3"},{"pattern": "(\\d{4})(\\d{3})(\\d{4})","leadingDigits": "1","format": "$1 $2 $3"},{"pattern": "(\\d{4})(\\d{1,2})(\\d{3})(\\d{4})","leadingDigits": "1","format": "$1 $2 $3 $4"}]},"generalDesc": {"nationalNumberPattern": "(?:[2-7]|9\\d)\\d{8}|2\\d{5}|(?:1800|8)\\d{7,9}"},"fixedLine": {"possibleLengths": {"national": "6,[8-10]","localOnly": "4,5,7"},"exampleNumber": "232345678","nationalNumberPattern": "(?:(?:2[3-8]|3[2-68]|4[2-9]|5[2-6]|6[2-58]|7[24578])\\d{3}|88(?:22\\d\\d|42))\\d{4}|(?:2|8[2-8]\\d\\d)\\d{5}"},"mobile": {"possibleLengths": {"national": "10"},"exampleNumber": "9051234567","nationalNumberPattern": "(?:8(?:1[37]|9[5-8])|9(?:0[5-9]|1[0-24-9]|[235-7]\\d|4[2-9]|8[135-9]|9[1-9]))\\d{7}"},"tollFree": {"possibleLengths": {"national": "[11-13]"},"exampleNumber": "180012345678","nationalNumberPattern": "1800\\d{7,9}"}},{"id": "PK","countryCode": "92","internationalPrefix": "00","nationalPrefix": "0","mobileNumberPortableRegion": "true","availableFormats": {"numberFormat": [{"pattern": "(\\d{3})(\\d{3})(\\d{2,7})","nationalPrefixFormattingRule": "$NP$FG","leadingDigits": "[89]0","format": "$1 $2 $3"},{"pattern": "(\\d{4})(\\d{5})","leadingDigits": "1","format": "$1 $2"},{"pattern": "(\\d{3})(\\d{6,7})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": ["2(?:3[2358]|4[2-4]|9[2-8])|45[3479]|54[2-467]|60[468]|72[236]|8(?:2[2-689]|3[23578]|4[3478]|5[2356])|9(?:2[2-8]|3[27-9]|4[2-6]|6[3569]|9[25-8])","9(?:2[3-8]|98)|(?:2(?:3[2358]|4[2-4]|9[2-8])|45[3479]|54[2-467]|60[468]|72[236]|8(?:2[2-689]|3[23578]|4[3478]|5[2356])|9(?:22|3[27-9]|4[2-6]|6[3569]|9[25-7]))[2-9]"],"format": "$1 $2"},{"pattern": "(\\d{2})(\\d{7,8})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits": "(?:2[125]|4[0-246-9]|5[1-35-7]|6[1-8]|7[14]|8[16]|91)[2-9]","format": "$1 $2"},{"pattern": "(\\d{5})(\\d{5})","nationalPrefixFormattingRule": "($NP$FG)","leadingDigits"
```

Comment:
Maybe it has to be a single line...

But I wonder if this would be a good time to format this file, so it's easier to read/parse in the future?